static/css/artist/artist_list.css

```diff
@@ -70,3 +73,35 @@
 .nav-float {
     float: left;
 }
+
+.artist-name {
+    padding-top: 5px;
+    padding-bottom: 0px;
+    font-family: Dosis, sans-serif;
+    background-color: #ecf0f1;
+}
+
+.artist-location {
+    padding-top: 5px;
+    padding-bottom: 10px;
+    font-family: Dosis, sans-serif;
```

Comment:
Yeah, I think it makes sense to just make a new class for all of the common attributes: font, background, etc. and apply both classes in each case.